docker/flink/Dockerfile

```
@@ -1,3 +1,3 @@
-FROM flink:1.18.0
+FROM flink:1.18.1
 # ibis-flink requires PyFlink dependency
 RUN wget -nv -P $FLINK_HOME/lib/ https://repo1.maven.org/maven2/org/apache/flink/flink-python/1.18.0/flink-python-1.18.0.jar
```

```suggestion
RUN wget -nv -P $FLINK_HOME/lib/ https://repo1.maven.org/maven2/org/apache/flink/flink-python/1.18.1/flink-python-1.18.1.jar
```
A little surprised it didn't cause any problems as is.